.gitlab/notify/notify.yml

```
@@ -91,7 +91,8 @@ notify_github:
 notify_gitlab_ci_changes:
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
   stage: notify
-  needs: []
+  needs: [compute_gitlab_ci_config]
+  dependencies: [compute_gitlab_ci_config]
```

If I'm correct the artifact of the `needs` job list is [already available by default](https://docs.gitlab.com/ee/ci/yaml/index.html#needsartifacts)
```suggestion
  needs: [compute_gitlab_ci_config]
```